docker-compose.yml

```diff
@@ -2,9 +2,7 @@ version: "3"
 
 services:
   postgres:
-    build:
-      context: ./docker/
-      dockerfile: ./plv8.dockerfile
+    image: bryanchriswhite/ledger-subquery-postgres:latest
```

Comment:
Temporary until `gcr.io/fetch-ai-images/subquery-postgres:863ce7d` is made public